### PR TITLE
Fixed broken URL, added an announcement on the top and an empty Robot.txt

### PR DESCRIPTION
**Pull request by pavangudiwada**

### CODE REVIEW

File: docs/_templates/base.html [hunk 1]
```diff
@@ -35,3 +35,8 @@
   }, false);
 </script>
 {% endblock %}
+{% block announce %}
```
Comment (aantn) [inline docs/_templates/base.html:38]: Can you split this to a separate PR and ask Keren for feedback on colors?

**Review by aantn — APPROVED** [review]

File: docs/_templates/base.html (hunk 1 above)
Comment (pavangudiwada) [reply docs/_templates/base.html]: Done
